docs/source/installation.rst

```diff
@@ -67,6 +67,44 @@ Using GitHub Container Registry
 
    docker pull ghcr.io/geopython/pygeoapi:latest   
 
+Kubenetes
```

Comment:
Is this file intended as part of this PR?

Reply:
Yes - I'll update the title. 

We think there's a bug when running on kubenetes with the connection pooling as each request could go to a separate replica and use a new connection

Reply:
We'll solve https://github.com/BritishGeologicalSurvey/pygeoapi/issues/27 asap